### PR TITLE
Post by Zarina on use of colors in Locus Tempus

### PR DESCRIPTION
**Pull request by zmustapha**

### CODE REVIEW

File: content/articles/ux-colors.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+poster_sourceurl: "https://unsplash.com/photos/4N4DiBz9c5s"
+poster_source: "Sanath Kumar on Unsplash.com"
+bookendanimal: "palette"
+---
+
+Software development in educational technologies benefits greatly from user experience design (UXD)
+to ensure that the user experience leads to the outcome of the pedagogical goals behind the
+application design.
+
+Color choices in user interface design is an important component of UXD. They can be a source of
```
Comment (sdreher) [inline content/articles/ux-colors.md:26]: 'are'

File: content/articles/ux-colors.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+{{< figure
+    src="/img/assets/ux-colors-lt-bw.png"
+    width="50%"
+    class="mx-auto bordered"
+    alt="Locus Tempus user interface in grayscale."
+>}}
+
+## Colors for user interface
+
+Looking at the palettes in map aforementioned mapping applications, blue is used to represent
```
Comment (sdreher) [inline content/articles/ux-colors.md:99]: "Looking at the palettes in `the` aforementioned

File: content/articles/ux-colors.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+    <small>AMBER #ffecb3</small>
+    </div>
+    <div class="col-2 p-2 py-3 p-md-3" style="line-height: 1; background: #fff8e1;">
+    <small>AMBER #fff8e1</small>
+    </div>
+</div>
+
+* Amber is warm, inviting, playful, and it focuses attention.
+* In maps, it represents a way of getting somewhere, and building blocks of interest. In Locus
+Tempus, amber is used to group parts and actions that lead the completion of an activity or action.
```
Comment (sdreher) [inline content/articles/ux-colors.md:152]: "lead to"

**Review by sdreher — COMMENTED** [review]
Just a few typos. Really excellent post Zarina!

File: content/articles/ux-colors.md (hunk 2 above)
Comment (zmustapha) [reply content/articles/ux-colors.md:99]: Fixed

File: content/articles/ux-colors.md (hunk 1 above)
Comment (zmustapha) [reply content/articles/ux-colors.md:26]: fixed

File: content/articles/ux-colors.md (hunk 3 above)
Comment (zmustapha) [reply content/articles/ux-colors.md:152]: fixed

**Pull request merged**